build-info-extractor-gradle/src/test/java/org/jfrog/gradle/plugin/artifactory/Utils.java

```diff
@@ -189,15 +190,24 @@ private static void assertRequestedBy(Build buildInfo) {
      * @return build info or null
      * @throws IOException - In case of any IO error
      */
-    private static Build getBuildInfo(ArtifactoryManager artifactoryManager, BuildResult buildResult) throws IOException {
+    public static Build getBuildInfo(ArtifactoryManager artifactoryManager, BuildResult buildResult) throws IOException {
+        Pair<String,String> buildDetails = getBuildDetails(buildResult);
+        return artifactoryManager.getBuildInfo(buildDetails.getLeft(), buildDetails.getRight(), null);
+    }
+
+    /**
+     * @param buildResult Details of the build
+     * @return A pair of: Left item - buildResult's build name, Right item - buildResult's build number
+     */
+    public static Pair<String,String> getBuildDetails( BuildResult buildResult)  {
```

Comment:
Please run reformat on this file
```suggestion
    public static Pair<String, String> getBuildDetails(BuildResult buildResult)  {
```